packages/actor-rdf-resolve-quad-pattern-federated/lib/ActorRdfResolveQuadPatternFederated.ts

```diff
@@ -16,12 +18,52 @@ import { FederatedQuadSource } from './FederatedQuadSource';
 export class ActorRdfResolveQuadPatternFederated extends ActorRdfResolveQuadPatternSource
   implements IActorRdfResolveQuadPatternFederatedArgs {
   public readonly mediatorResolveQuadPattern: MediatorRdfResolveQuadPattern;
+  public readonly mediatorRdfMetadataAccumulate: MediatorRdfMetadataAccumulate;
   public readonly skipEmptyPatterns: boolean;
 
   protected readonly emptyPatterns: Map<IDataSource, RDF.Quad[]> = new Map();
 
   public constructor(args: IActorRdfResolveQuadPatternFederatedArgs) {
     super(args);
+
+    // TODO: remove this backwards-compatibility in the next major version, and make the param mandatory
```

Comment:
I think keeping it in the source code is fine, otherwise we have the same list in 2 places (and they are bound to get out of sync).
I always go through all TODO's upon every major update.